### ISSUE TITLE
add training

### ISSUE THREAD

**Issue by kingfischer16** [kingfischer16/srcnn-tf2#4]
Add functions for training and benchmarking. Should cover the basic training and benchmarking found in the paper, as well as make room for more advanced training (other loss functions, other models).

**Comment by kingfischer16** [comment]
Completed with PR #2
